backend/src/test/java/com/funeat/recipe/application/RecipeServiceTest.java

```diff
@@ -73,6 +79,39 @@ class RecipeServiceTest {
                 .isEqualTo(expected);
     }
 
+    @Test
+    void 레시피의_상세_정보를_조회할_수_있다() {
+        // given
+        final var category = 카테고리_추가_요청(new Category("간편식사", CategoryType.FOOD));
+        final var product1 = 상품_추가_요청(new Product("불닭볶음면", 1000L, "image.png", "엄청 매운 불닭", category));
+        final var product2 = 상품_추가_요청(new Product("참치 삼김", 2000L, "image.png", "담백한 참치마요 삼김", category));
+        final var product3 = 상품_추가_요청(new Product("스트링 치즈", 1500L, "image.png", "고소한 치즈", category));
+        final var recipeAuthor = 멤버_추가_요청(new Member("author", "image.png", "1"));
```

Comment:
```suggestion
        final var product3 = 상품_추가_요청(new Product("스트링 치즈", 1500L, "image.png", "고소한 치즈", category));
        
        final var author = 멤버_추가_요청(new Member("author", "image.png", "1"));
        final var authorId = author.getId();
```

이거는 어떤가요? `authorId`는 밑에 when절에서 쓰여요!